tools/openshift-ci/Dockerfile

```diff
@@ -37,6 +43,13 @@ RUN set -x && \
     yum clean all -y && \
     rm -rf /var/cache/yum /var/tmp/* /tmp/*
 
+#install ansible 
+RUN set -x && \
```

Comment:
Do not invent new, append this block to the last `RUN` step to maintain the same style